proposals/dao/fip_60.ts

```diff
@@ -0,0 +1,255 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { deploy as deploySTW } from '@scripts/deploy/deployStakingTokenWrapper';
+import { getImpersonatedSigner } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+chai.use(CBN(ethers.BigNumber));
+
+const UNDERLYINGS = [
+  '0xBaaa1F5DbA42C3389bDbc2c9D2dE134F5cD0Dc89',
+  '0x06cb22615BA53E60D67Bf6C341a0fD5E718E1655',
+  '0x3D1556e84783672f2a3bd187a592520291442539'
+];
+const NAMES = ['FeiRari d3pool Fuse', 'FeiRari FEI-3Crv Metampool Fuse', 'FeiRari Gelato FEI-DAI LP Fuse'];
+const SYMBOLS = ['fD3-8', 'fFEI-3Crv-8', 'fG-UNI-FEI-DAI-8'];
+
+const COLLATERAL_FACTOR = ethers.constants.WeiPerEther.mul(60).div(100); // 60% CF
+
+const IMPL = '0x67Db14E73C2Dce786B5bbBfa4D010dEab4BBFCF9';
+const BECOME_IMPL_DATA = [];
+
+const COMPTROLLER = '0xc54172e34046c1653d1920d40333dd358c7a1af4'; // Pool 8 comptroller
+const IRM = '0xbab47e4b692195bf064923178a90ef999a15f819'; // ETH IRM (not really used because borrowing off)
+
+/*
+FIP-60: Add tokens to FeiRari
+
+DEPLOY ACTIONS:
+
+1. Deploy TribalChiefSyncExtension
+2. Deploy StakedTokenWrapper d3
+3. Deploy StakedTokenWrapper FEI-3Crv
+4. Deploy StakingTokenWrapper Gelato
+5. Deploy AutoRewardsDistributor d3
+6. Deploy AutoRewardsDistributor FEI-3Crv
+7. Deploy AutoRewardsDistributor Gelato
+
+OA ACTIONS:
+1. Add cTokens
+2. Set new oracle on Comptroller
+3. Add rewards to TribalChief
+4. Grant AutoRewardsDistributor role to ARDs on RewardsDistributorAdmin
+*/
+
+const setMaster = async function (addresses, newOracle) {
+  const comptroller = await ethers.getContractAt('Unitroller', addresses.rariPool8Comptroller);
+
+  const admin = await comptroller.admin();
+  await forceEth(admin);
+  const signer = await getImpersonatedSigner(admin);
+
+  await comptroller.connect(signer)._setPriceOracle(newOracle);
+
+  console.log(await comptroller.oracle());
+};
+
+const deployCToken = async function (addresses, underlying, name, symbol) {
+  const comptroller = await ethers.getContractAt('Unitroller', addresses.rariPool8Comptroller);
+
+  const admin = await comptroller.admin();
+  await forceEth(admin);
+  const signer = await getImpersonatedSigner(admin);
+
+  const calldata = ethers.utils.defaultAbiCoder.encode(
```

Comment:
I'd much prefer if we pulled the abi/interface from an import than inline it here, we'll make a mistake doing this eventually and not realize it

Reply:
in general yeah but we don't have this contract in the repo and I think this is about as error prone as rolling our own interface

Reply:
We could add the contract to a new folder, say contracts/external or whatever. For future.